wiki/streaming-chat-completions-route.md

```
@@ -0,0 +1,125 @@
+    "messages": [
+      { "role": "user", "content": "Tell me a story" }
+    ],
+    "stream": true
+  }'
+```
+
+The `-N` flag disables buffering so you can see chunks arrive in real time.
+
+The server responds with a stream of events, each prefixed with `data:`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)